src/server/handlers/thirdpartyRequests/transactions/{ID}/authorizations.ts

```diff
@@ -62,12 +63,8 @@ export async function post (
   request: Request,
   h: ResponseToolkit): Promise<ResponseObject> {
   // Hapi-OpenAPI plugin validates the payload schema for
-  // existence of all properties and their value types according
-  // to the OpenAPI specification. It also ensures non-null
-  // payload values.
-
-  // Does it automatically return error response for bad schema?
-  // Need to test?
+  // existence of properties and their types based on the
+  // OpenAPI specification. It also ensures non-null values.
 
   const payload: AuthPayload = request.payload as AuthPayload
 
```

Comment:
As discussed yesterday during a standup with @jgeewax  please move expensive mySql related semantic validation to background and return 202 asap -